style_guides/pl_sql_tables.md

```diff
@@ -0,0 +1,64 @@
+# Database Tables
+
+## Table Names
+
+Tables should be given a meaningful name that describes the purpose of the table. All table names should be plural and where the table name contains more than one word only the last should be plural. 
+
+Table names should be underscore separated. 
+

```

Comment:
Can we add in an example here for "normal" table names. My eye is looking for an example and jumps to the one below for shared database schema 

```
programmes
programme_years
programme_year_modules
```